go/vt/vttablet/tabletserver/gc/tablegc.go

```diff
@@ -210,101 +194,113 @@ func (collector *TableGC) Open() (err error) {
 		delete(collector.lifecycleStates, schema.EvacTableGCState)
 	}
 
+	ctx := context.Background()
+	ctx, collector.cancelOperation = context.WithCancel(ctx)
+	go collector.operate(ctx)
```

Comment:
moving operate from init to Open. Does this means GC will start later then what it is currently now. Do you see any issue because of that delay?

Reply:
I don't see an issue, because `Operate()` works on intervals anyhow. There is no guarantee when exactly the cycles begin. The default cycle interval is `1h`. So the precise timing of when the cycle begins is insignificant.